config/azkaban/exec-server/start-exec.sh

```diff
@@ -3,6 +3,6 @@
 script_dir=$(dirname $0)
 
 echo "Creating tables in database if they do not exist using create-all-sql.sql"
-mysql -h ${db_host} -u${db_username} -p${db_password} ${db_name} < /azkaban-db/create-all-sql-0.1.0-SNAPSHOT.sql
+mysql -h $DB_HOST -u$DB_USERNAME -p$DB_PASSWORD $DB_NAME < /azkaban-db/create-all-sql-0.1.0-SNAPSHOT.sql
```

Comment:
Surely these env vars will be empty since its the lowercase env vars that are defined in the template files azkaban_webserver_s3.tf and azkaban_executor_s3.tf

Reply:
They are exported during container startup. As the secrets are rotated they are not defined in terraform.